run/README.namelist

```diff
@@ -1068,6 +1068,12 @@ Namelist variables for lake module:
                                                   the lake depth data, but this switch is set to 1, the program will stop and tell one to go back to geogrid
                                                   program. 
                                                   = 0, do not use lake depth data.
+ shalwater_rough                     = 1,       ! Shallow water roughness scheme on/off.
+                                                  Bathymetry dataset from GEBCO Compilation Group. Please acknowledge the following in presentations and 
+                                                  publications: GEBCO Compilation Group (2021) GEBCO 2021 Grid (doi:10.5285/c6612cbe-50b3-0cff-e053-6c86abc09f8f)
+ shalwater_depth                     = 0.0,     ! Set constant depth [m] for shallow water roughness scheme if no bathymetry data availabile.
+                                                  The scheme is intended for depths between 10.0 and 100.0 m. Any depths outside of this range will be
+                                                  rounded to the nearest limit value.
```

Comment:
@phawbeck Can you add the option values and their definition here (< 0, > 0, etc.)?

Reply:
I'm not sure what else to include for this - shalwater_rough is either 0 or 1 (off/on), and shalwater_depth is a constant depth in meters that will be set for the entire water_depth field. So, if shalwater_depth is less than 10 m, then it will be set to 10 m. If it is larger than 100 m, then it will be set to 100 m. Does that make sense or should I re-word the description?

Reply:
@phawbeck What does negative value mean, for example?

Reply:
Ah ok, I will add a statement for negative values (shalwater_depth must be positive).